astropy/cosmology/funcs.py

```diff
@@ -66,12 +66,13 @@ def z_at_value(func, fval, zmin=1e-8, zmax=1000, ztol=1e-8, maxfun=500):
     in a Planck13 cosmology, you could do the following:
 
     >>> import astropy.units as u
-    >>> from astropy.cosmology import Planck13, z_at_value
+    >>> from astropy.cosmology import z_at_value
+    >>> from astropy.cosmology.parameters import Planck13
```

Comment:
The import will work now, but the following tests try to use the `distmod` and `age` properties (which the dict doesn't have).